### DISCUSSION TITLE
What's the user of postinstall.sh in blob-fuse2?

### DISCUSSION THREAD

**Discussion by raj-prince** [Azure/azure-storage-fuse#1074] [General]
As we keep the postinstall.sh in /usr/share/blobfuse2 folder.

Does anyone know, what's the use case of this file? 

When does this file get executed?

Thanks,
Prince.

**Comment by bhansalivikas** [comment]
There were earlier preview versions of Blobfuse which used this file to set up auto shell completion and some syslog filters. These files are no longer used and latest installers do not copy these files as well.
